### PR TITLE
Add .bat Files for Windows Use

### PR DESCRIPTION
**Pull request by Tom271**
To be able use timetrack on Windows, I've created some quick .bat files to replace the .sh file for Unix systems and added a section to the README. This is my first time writing batch scripts so there will definitely be some improvements to be made! Just in case they are useful for others. I also had to remove the `src.` from the `from src.single_calendar ..., from src.plotting ... , from src.helpers ..` lines in `multi_calendar.py` and `single_calendar.py`. I've left that off this pull request as I can't test if the same change is necessary on Mac.

Thanks for creating a great tool!

### CODE REVIEW

File: timetrack_single_cal.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@echo OFF
+SET %TIMETRACKDIR=\Users\nicole\code\timetrack
```
Comment (wuningxi) [inline timetrack_single_cal.bat]: Add `SET PREVDIR=%CD%` here to save the previous working directory

File: timetrack_single_cal.bat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+:timetrack_single_cal
+CALL cd %TIMETRACKDIR%
+ECHO %CD%
+CALL python src\single_calendar.py %*
```
Comment (wuningxi) [inline timetrack_single_cal.bat:7]: add `CALL cd %PREVDIR%` to go back to the previous working directory

File: timetrack_multi_cal.bat [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@echo OFF
+SET %TIMETRACKDIR=\Users\nicole\code\timetrack
+
+:timetrack_multi_cal
+CALL cd %TIMETRACKDIR%
+ECHO %CD%
```
Comment (wuningxi) [inline timetrack_multi_cal.bat]: The print statement is no longer necessary if we switch back to the previous working directory

File: timetrack_multi_cal.bat [hunk 4]
```diff
@@ -0,0 +1,9 @@
+@echo OFF
+SET %TIMETRACKDIR=\Users\nicole\code\timetrack
```
Comment (wuningxi) [inline timetrack_multi_cal.bat]: Add `SET PREVDIR=%CD%` here to save the previous working directory

File: timetrack_single_cal.bat [hunk 5]
```diff
@@ -0,0 +1,9 @@
+@echo OFF
+SET %TIMETRACKDIR=\Users\nicole\code\timetrack
+
+:timetrack_single_cal
+CALL cd %TIMETRACKDIR%
+ECHO %CD%
```
Comment (wuningxi) [inline timetrack_single_cal.bat]: The print statement is no longer necessary if we switch back to the previous working directory

File: timetrack_multi_cal.bat [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+:timetrack_multi_cal
+CALL cd %TIMETRACKDIR%
+ECHO %CD%
+CALL python src\multi_calendar.py %*
```
Comment (wuningxi) [inline timetrack_multi_cal.bat:7]: add CALL cd %PREVDIR% to go back to the previous working directory

File: README.md [hunk 7]
```diff
@@ -149,3 +149,11 @@ optional arguments:
...
 ```
 
+## Use on Windows
+
+- To use on Windows, change '%TIMETRACKDIR' to the location of your timetrack folder in both `timetrack_multi_cal.bat` and `timetrack_single_cal.bat`.
+- Add the timetrack folder to the system path environment variables. This allows the use of `timetrack_multi_cal` and `timetrack_single_cal` from anywhere in the command line. On Windows 10, this is done via `Control Panel > System and Security > System > Advanced System Settings > Environment Variables`. Then double click on `Path` in System Variables then the `Browse...` button.
+
+All other setup and usage will be the same as above. How you export the .ics files depends on your calendar client.
+
+Note that these functions will *not* return you to the directory you began in but will move you to the timetrack folder.
```
Comment (wuningxi) [inline README.md]: The last line can be deleted with proposed changes.

File: README.md [hunk 8]
```diff
@@ -149,3 +149,11 @@ optional arguments:
 
 ```
 
+## Use on Windows
+
+- To use on Windows, change '%TIMETRACKDIR' to the location of your timetrack folder in both `timetrack_multi_cal.bat` and `timetrack_single_cal.bat`.
+- Add the timetrack folder to the system path environment variables. This allows the use of `timetrack_multi_cal` and `timetrack_single_cal` from anywhere in the command line. On Windows 10, this is done via `Control Panel > System and Security > System > Advanced System Settings > Environment Variables`. Then double click on `Path` in System Variables then the `Browse...` button.
```
Comment (wuningxi) [inline README.md]: Thanks for adding these instructions regarding the Windows setup. Could you move them further up to `## Setup` (maybe under a separate `### Windows` section), so they're easier to find?

**Review by wuningxi — CHANGES_REQUESTED** [review]
Thanks for porting the project to Windows, Tom! It looks good. I've added some comments for moving the Windows instructions in `README.md` and changing back to the previous working directory after executing the python scripts as this might be more convenient for users. Regarding the removal of `src.`, you can add this to the pull request as well, as it will still work on Mac.

**Comment by Tom271** [comment]
Thanks for the review, I've made all the changes you suggested and updated the PR.

**Review by wuningxi — APPROVED** [review]
Thanks for the changes 👍

**Comment by wuningxi** [comment]
I'm pulling the changes into the `windows-support` branch just to make sure everything is working fine before merging with the main branch.

**Pull request merged**